lucent/optvis/render.py

```diff
@@ -81,6 +114,20 @@ def render_vis(
 
     transform_f = transform.compose(transforms)
 
+    # TODO this only works for optimisation in Fourier Space now, support pixel space as well
```

Comment:
Add a check here that throws an error if a non-fourier parametrization is used.

Reply:
did it by simply checking the dimensionality of params, which seems unelegant, but don't see a nicer way...